### ISSUE TITLE
modal popup appears to be running JavaScript entered in the comment

### ISSUE THREAD

**Issue by ploray** [zendesklabs/oops_app#24]
It appears that any JavaScript entered into a ticket comment is run by the modal prior to ticket submission.  For example, when attempting to submit this comment:

```
Hello, again!

Your conversion pixel would actually look like the following:

<script>
ga('send', 'event', 'X', 'Y', 'Z'); 
</script>

Hope this helps! Let me know if you need any further assistance.
```

The modal will run that JavaScript and prevent the response from being submitted due to an app error:

![screenshot_122016_122144_pm](https://cloud.githubusercontent.com/assets/14320872/21361226/1bc64334-c6b1-11e6-841e-5e5867d34d94.jpg)

Tried using markdown to format the JS in a code block, but didn't appear to make any difference.  Disabling the app resolves the issue.

**Comment by dpawluk** [comment]
@ploray thanks so much, this definitely should not be happening. I'll keep you posted on the outcome.

**Comment by JoshGlazebrook** [comment]
This should be fixed here once accepted: #28
